.github/workflows/run-tests.yml

```
@@ -0,0 +1,37 @@
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      # Step 2: Set up Deno
+      - name: Set up Deno
+        uses: denoland/setup-deno@v2
+        with:
+          deno-version: v2.x
+
+      # Step 3: Install dependencies (if needed) and cache them
+      - name: Cache dependencies
+        uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)